meta-mender-demo/conf/layer.conf

```diff
@@ -8,6 +8,10 @@ BBPATH .= ":${LAYERDIR}"
 BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 	${LAYERDIR}/recipes-*/*/*.bbappend"
 
+BBFILES_DYNAMIC += "\
+  mender-commercial:${LAYERDIR}/recipes-*/*/mender-monitor*.bbappend \
```

Comment:
This still doesn't work because we need to exclude, somehow, these recipes from the `BBFILES +=` above. I get:

```
ERROR: No recipes available for:
  /home/lluis/poky/meta-mender/meta-mender-demo/recipes-mender/mender-monitor/mender-monitor_%.bbappend
```

I don't know the best way to do it, though. Easy suggestion: we have an explicit directory for the recipes belonging to `mender-commercial` collection, like:
```
meta-mender-demo/recipes-mender/...                                    <- bbappends to core recipes
meta-mender-demo/mender-commercial/recipes-mender/...   <- bbappends to commercial recipes
```

This way they are exluded in `BBFILES` and can easily be included on DYNAMIC with:
```
BBFILES_DYNAMIC += "\
  mender-commercial:${LAYERDIR}/mender-commercial/recipes-*/*/*.bbappend \
```

(extra advantage of this approach is that we don't need to further modify `layer.conf` when adding recipes there)

Reply:
Hmm, interesting, previously I did what you mentioned above, but then I did not want another repo, so I changed it back, and it worked for me.

Let me retest it.

Reply:
I sincerely thought it would mask them from BBFILES, but maybe I was fooled.

Reply:
Added to a separate directory `mender-commercial`